.env.example

```diff
@@ -4,5 +4,5 @@ DATABASE_URL=postgresql://postgres:postgres@localhost:5432/antiplag
 CELERY_BROKER_URL=pyamqp://
 CELERY_RESULT_BACKEND=rpc://
 CELERY_RESULT_PERSISTENT=True
-ELASTIC_HOST=elastic:password@localhost:9200
+ELASTIC_HOST=elastic:pNA4gPABp6p2TMnowIcN@elastic.antiplag.sk:80
```

Comment:
Please remove this change.

Also, it would be better if you changed the elastic password now and updated it in the docs - as it was once exposed to the internet.